frontend.bat

```diff
@@ -523,14 +523,18 @@ for /F "tokens=1,2 delims=." %%a in ("!vsver!") do (
     rem https://github.com/3F/hMSBuild/issues/3
     set vsver=%%~a.0
 )
+REM Workaround for VS2019, may use 'vswhere -find' instead
```

Comment:
My interpretation of the vswhere changes was that MS recommended that way instead (I do not see how though)

Reply:
I thought about implementing through new `-find` feature, but only as an additional way. Maybe via new key and so on. 

Three reasons: 

- Glob(wildcards) searching will work slower instead of direct path.
- This will work only for new vswhere, anyway.
- Mainly, this is just additional feature of vswhere for simplified working through their tool.


The core inside [the following function](https://github.com/3F/hMSBuild/blob/a40c6ac420708cbc86a2d93c9f374feeb42bb9cd/frontend.bat#L439) `:vswhereBin {in:vswbin} {out:toolset} {optin:vswcache}`

steps from [main logic](https://github.com/3F/hMSBuild/blob/a40c6ac420708cbc86a2d93c9f374feeb42bb9cd/frontend.bat#L286)

You can create issue for suggestion that should be considered later.

Reply:
Will change comment.
Then vswhere -find is new functionality, a separate (feature) issue could be added. I do not know how to target/angle it at the moment.